queries/cedar/highlights.scm

```diff
@@ -0,0 +1,69 @@
+; Function calls
+(call_expression
+  function: (selector_expression
+    field: (field_identifier) @function.method))
+
+[
+  "contains"
+  "containsAll"
+] @function
+
+; Annotations
+(annotation) @attribute
```

Comment:
This needs to be `(annotation (identifier @attribute))`; as of now it gets overridden by `@variable`. Then you'll also need a separate one for `(annotation "@" @attribute)`